prow/scripts/cluster-integration/helpers/kyma-cli.sh

```diff
@@ -8,8 +8,9 @@ install::kyma_cli() {
     pushd "${INSTALL_DIR}/bin"
 
     log::info "- Install kyma CLI ${os} locally to a tempdir..."
-
-    curl -sSLo kyma "https://storage.googleapis.com/kyma-cli-stable/kyma-${os}?alt=media"
+#TODO
+#    curl -sSLo kyma "https://storage.googleapis.com/kyma-cli-stable/kyma-${os}?alt=media"
```

Comment:
Kyma cli 1.10.0 is out now. We can revert that hack :)

Reply:
Reverted